wordpress-mobile/WordPress-iOS
# Blogging Prompts Dashboard Card: hide prompt card when prompt skipped

Ref: #18429, #18250

When `Skip this prompt` is selected from the prompt dashboard:
- The prompt information is stored in User Defaults.
- The card is removed from the Dashboard.

Bonus: the card will now use the cached prompt if there is one.

To test:
- Enable `bloggingPrompts` feature flag.
- Select the My Site `Home` tab.
- On the prompt card menu, select `Skip this prompt`.
- Verify the card is removed from the Dashboard.
- Select a different site.
- Verify the prompt card appears.

| ![prompt](https://user-images.githubusercontent.com/1816888/168909391-0250eda9-ca17-4880-8972-c74a2cc29dba.png) | ![no_prompt](https://user-images.githubusercontent.com/1816888/168909384-5dc088bd-42cd-4e80-a4ca-aad477c9988a.png) |
|--------|-------|

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
